server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -48,7 +48,7 @@ public PropertiesConfiguration getCassandraConfigurationForDocker() {
         configuration.addProperty("cassandra.keyspace", "apache_james");
         configuration.addProperty("cassandra.replication.factor", 1);
         configuration.addProperty("cassandra.retryConnection.maxRetries", 10);
-        configuration.addProperty("cassandra.retryConnection", 5000);
+        configuration.addProperty("cassandra.retryConnection.minDelay", 5000);
```

Comment:
Basically, in the CassandraSessionModule we just load the value of the key "cassandra.retryConnection.minDelay" for the specified value that we want to set. We do not load any value of the key "cassandra.retryConnection". 
So I think we will add the property file that we support and read.
I checked the history of cassandra.properties file and it does not contain the key "cassandra.retryConnection"